### ISSUE TITLE
Provide a way to specify "connection setup" SQL

### ISSUE THREAD

**Issue by nvoxland** [liquibase/liquibase#2008]
# Overview

Currently, the logic Liquibase uses to connect to the database is contained in the the Database implementations. 

But, sometimes people have specific custom SQL that they'd like to run as Liquibase is first connecting, such as:

```
SET quoted_identifier OFF;
set proxy dbMainUser;
set replication off;
```
from https://stackoverflow.com/questions/68501167/liquibase-spring-boot-databasechangelog-creation-failing

# Implementation Ideas

Perhaps a new configuration property like `liquibase.connection.initSql` which can contain SQL ran as part of the connection logic would work well? The changelog file isn't parsed until later in the process, so we can't store it in there.

Other ideas?

**Comment by ppo-05** [comment]
Good Idea, I also need to execute queries when liquibase connect to a snowflake account.
We could have **liquibase.connection.initSql** or also a **liquibase.connection.initSqlFile** to store multiple queries in a single file

**Comment by filipelautert** [comment]
We're currently triaging our backlog to ensure we can focus on the issues that matter most. With a large number of open issues, it becomes difficult to give each one the attention it deserves.

This issue has been closed due to inactivity — there have been no updates for over 30 days. If this is still relevant to you, please feel free to reopen it or create a new issue with updated context. We appreciate your contribution!
